docs/book/usage/started.md

```diff
@@ -1,8 +1,277 @@
-# Getting started
+# Getting Started with Walrus
 
-You want to get a taste of Walrus? Here are some sites that are built on and/or use Walrus:
+Walrus is a platform for building efficient and resilient data markets, where data is stored as
+*blobs*. Walrus stores all blobs as an array of bytes with a fixed size, and you can store any
+type of file, such as text, video, or source code.
 
-- [Mint a toy NFT](https://flatland.wal.app) on the flatland application.
-- [Browse scientific papers](https://snowreads.wal.app).
+Sui is a blockchain that supports programmability at a
+[fundamental level](https://docs.sui.io/concepts/transactions/prog-txn-blocks). Walrus binds all
+blobs to *objects* on the Sui blockchain.
 
-You can explore blobs that are stored on Walrus through the [Walruscan explorer](https://walruscan.com).
+## Walrus and Sui
+
+Walrus depends on Sui, as it leverages Sui to track blobs, their respective owners, and their
+lifetimes.
+
+Sui and Walrus are both decentralized, distributed systems made up of many independent servers
+that communicate and collectively establish shared state. A group of servers together is a
+*network*.
+
+### Available networks
+
+Sui and Walrus each have the following available networks:
+
+- **Testnet** is a sandbox-like network where you can receive test tokens for free to use for the
+  network's fees. You can build, test, and debug software packages on Testnet. Testnet does not
+  guarantee data persistence and might wipe data at any time without warning.
+- **Mainnet** is a production environment where you use real tokens and users or other
+  applications rely on consistent functionality.
+
+When you are getting started, you should use **Testnet.**
+
+## Install tooling
+
+To install Walrus and Sui, use the Mysten Labs `suiup` tool. First,
+[install suiup](https://github.com/MystenLabs/suiup?tab=readme-ov-file#installation):
+
+```bash
+curl -sSfL https://raw.githubusercontent.com/Mystenlabs/suiup/main/install.sh | sh
+```
+
+Then, install `sui` and `walrus`:
+
+```bash
+suiup install sui
+suiup install walrus
+```
+
+### Configure tooling for Walrus Testnet
+
+After you install Walrus, you need to configure the Walrus client, which tells it the RPC URLs to
+use to access Testnet or Mainnet. The easiest way to configure Walrus is to download the following
+prefilled configuration file.
+
+```bash
+mkdir ~/.config/walrus/
```

Comment:
Nit: As we use the `--create-dirs` option in the next line, this is not necessary.

```suggestion
```